src/components/ec-option-card/ec-option-card.vue

```diff
@@ -0,0 +1,199 @@
+<template>
+  <component
+    :is="componentTag"
+    v-bind="{
+      ...attrs,
+      ...componentProps,
+      'data-test': attrs['data-test'] ? `${attrs['data-test']} ec-option-card` : 'ec-option-card',
+    }"
+    class="ec-option-card"
+    :class="getOptionCardClasses()"
+    @click="setClickAction()"
+  >
+    <span
+      data-test="ec-option-card__title"
+    >
+      <ec-icon
+        v-if="optionCardIconName && optionTitle"
+        data-test="ec-option-card__title-img"
+        class="ec-option-card__title-img"
+        :class="getOptionIconClass()"
+        :name="optionCardIconName"
+        :alt="optionTitle"
+        :size="15"
+      />{{ optionTitle }}</span>
+    <p
+      v-if="optionCaption"
```

Comment:
do we have a variant with no caption?

Reply:
Yes, Only title. 
![image](https://user-images.githubusercontent.com/15978652/220640092-4a0dd557-056b-4734-a0f9-59a4b6fc3d00.png)
https://www.figma.com/file/qLv2in1OHCrSOEoNivIQA1/Guidelines?node-id=893%3A22867&t=bbJwWHDtlJeQFm2p-0